src/stories/0-Welcome.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
- [ ] This is a good thought but let's only create stories that are related to our components